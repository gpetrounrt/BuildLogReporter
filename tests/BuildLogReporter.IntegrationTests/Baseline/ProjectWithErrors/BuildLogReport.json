{
  "error_count": 2,
  "warning_count": 0,
  "log_entries": [
    {
      "type": 0,
      "code": "CS0029",
      "message": "Cannot implicitly convert type \u0027string\u0027 to \u0027int\u0027",
      "file_path": "\\TestSupportProjects\\ProjectWithErrors\\Program.cs",
      "line_number": 1
    },
    {
      "type": 0,
      "code": "CS0029",
      "message": "Cannot implicitly convert type \u0027int\u0027 to \u0027string\u0027",
      "file_path": "\\TestSupportProjects\\ProjectWithErrors\\Program.cs",
      "line_number": 2
    }
  ]
}