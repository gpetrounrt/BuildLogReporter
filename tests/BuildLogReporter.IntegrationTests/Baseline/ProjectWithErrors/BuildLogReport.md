Errors: 2, Warnings: 0

| Type | Code | Message | File path | Line number |
|:---|:---|:---|:---|:---|
| Error | CS0029 | Cannot implicitly convert type 'string' to 'int' | \TestSupportProjects\ProjectWithErrors\Program.cs | 1 |
| Error | CS0029 | Cannot implicitly convert type 'int' to 'string' | \TestSupportProjects\ProjectWithErrors\Program.cs | 2 |
