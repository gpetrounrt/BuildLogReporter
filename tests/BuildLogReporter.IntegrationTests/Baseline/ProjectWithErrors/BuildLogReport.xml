<?xml version="1.0" encoding="utf-8"?>
<ProcessedLogResult ErrorCount="2" WarningCount="0">
  <LogEntries>
    <LogEntry Type="Error" Code="CS0029" Message="Cannot implicitly convert type 'string' to 'int'" FilePath="\TestSupportProjects\ProjectWithErrors\Program.cs" LineNumber="1" />
    <LogEntry Type="Error" Code="CS0029" Message="Cannot implicitly convert type 'int' to 'string'" FilePath="\TestSupportProjects\ProjectWithErrors\Program.cs" LineNumber="2" />
  </LogEntries>
</ProcessedLogResult>