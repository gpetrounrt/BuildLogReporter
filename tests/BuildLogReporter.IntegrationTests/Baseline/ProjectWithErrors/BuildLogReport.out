Starting processing of '\ProjectWithErrors\output.binlog'...
Starting extraction of log entries...
Opening binary log file...
Opened binary log file.
Processing records...
Found error record.
Type: Error
Code: CS0029
Message: Cannot implicitly convert type 'string' to 'int'
File path: \TestSupportProjects\ProjectWithErrors\Program.cs
Line Number: 1
Found error record.
Type: Error
Code: CS0029
Message: Cannot implicitly convert type 'int' to 'string'
File path: \TestSupportProjects\ProjectWithErrors\Program.cs
Line Number: 2
Processed N record(s).
Completed extraction in elapsed time.
Found 2 error(s) and 0 warning(s).
Generating Badge report...
Generated report in elapsed time.
Saving report to \ProjectWithErrors.
Saved report.
Generating Html report...
Generated report in elapsed time.
Saving report to \ProjectWithErrors.
Saved report.
Generating Json report...
Generated report in elapsed time.
Saving report to \ProjectWithErrors.
Saved report.
Generating Markdown report...
Generated report in elapsed time.
Saving report to \ProjectWithErrors.
Saved report.
Generating Xml report...
Generated report in elapsed time.
Saving report to \ProjectWithErrors.
Saved report.
Completed processing in elapsed time.
